<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />

<meta name="viewport" content="width=device-width, initial-scale=1">

<meta name="author" content="Brittany N. Lasseigne, PhD and Ryne C. Ramaker" />

<meta name="date" content="2017-02-16" />

<title>ProliferativeIndex Vignette</title>



<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>



<link href="data:text/css;charset=utf-8,body%20%7B%0Abackground%2Dcolor%3A%20%23fff%3B%0Amargin%3A%201em%20auto%3B%0Amax%2Dwidth%3A%20700px%3B%0Aoverflow%3A%20visible%3B%0Apadding%2Dleft%3A%202em%3B%0Apadding%2Dright%3A%202em%3B%0Afont%2Dfamily%3A%20%22Open%20Sans%22%2C%20%22Helvetica%20Neue%22%2C%20Helvetica%2C%20Arial%2C%20sans%2Dserif%3B%0Afont%2Dsize%3A%2014px%3B%0Aline%2Dheight%3A%201%2E35%3B%0A%7D%0A%23header%20%7B%0Atext%2Dalign%3A%20center%3B%0A%7D%0A%23TOC%20%7B%0Aclear%3A%20both%3B%0Amargin%3A%200%200%2010px%2010px%3B%0Apadding%3A%204px%3B%0Awidth%3A%20400px%3B%0Aborder%3A%201px%20solid%20%23CCCCCC%3B%0Aborder%2Dradius%3A%205px%3B%0Abackground%2Dcolor%3A%20%23f6f6f6%3B%0Afont%2Dsize%3A%2013px%3B%0Aline%2Dheight%3A%201%2E3%3B%0A%7D%0A%23TOC%20%2Etoctitle%20%7B%0Afont%2Dweight%3A%20bold%3B%0Afont%2Dsize%3A%2015px%3B%0Amargin%2Dleft%3A%205px%3B%0A%7D%0A%23TOC%20ul%20%7B%0Apadding%2Dleft%3A%2040px%3B%0Amargin%2Dleft%3A%20%2D1%2E5em%3B%0Amargin%2Dtop%3A%205px%3B%0Amargin%2Dbottom%3A%205px%3B%0A%7D%0A%23TOC%20ul%20ul%20%7B%0Amargin%2Dleft%3A%20%2D2em%3B%0A%7D%0A%23TOC%20li%20%7B%0Aline%2Dheight%3A%2016px%3B%0A%7D%0Atable%20%7B%0Amargin%3A%201em%20auto%3B%0Aborder%2Dwidth%3A%201px%3B%0Aborder%2Dcolor%3A%20%23DDDDDD%3B%0Aborder%2Dstyle%3A%20outset%3B%0Aborder%2Dcollapse%3A%20collapse%3B%0A%7D%0Atable%20th%20%7B%0Aborder%2Dwidth%3A%202px%3B%0Apadding%3A%205px%3B%0Aborder%2Dstyle%3A%20inset%3B%0A%7D%0Atable%20td%20%7B%0Aborder%2Dwidth%3A%201px%3B%0Aborder%2Dstyle%3A%20inset%3B%0Aline%2Dheight%3A%2018px%3B%0Apadding%3A%205px%205px%3B%0A%7D%0Atable%2C%20table%20th%2C%20table%20td%20%7B%0Aborder%2Dleft%2Dstyle%3A%20none%3B%0Aborder%2Dright%2Dstyle%3A%20none%3B%0A%7D%0Atable%20thead%2C%20table%20tr%2Eeven%20%7B%0Abackground%2Dcolor%3A%20%23f7f7f7%3B%0A%7D%0Ap%20%7B%0Amargin%3A%200%2E5em%200%3B%0A%7D%0Ablockquote%20%7B%0Abackground%2Dcolor%3A%20%23f6f6f6%3B%0Apadding%3A%200%2E25em%200%2E75em%3B%0A%7D%0Ahr%20%7B%0Aborder%2Dstyle%3A%20solid%3B%0Aborder%3A%20none%3B%0Aborder%2Dtop%3A%201px%20solid%20%23777%3B%0Amargin%3A%2028px%200%3B%0A%7D%0Adl%20%7B%0Amargin%2Dleft%3A%200%3B%0A%7D%0Adl%20dd%20%7B%0Amargin%2Dbottom%3A%2013px%3B%0Amargin%2Dleft%3A%2013px%3B%0A%7D%0Adl%20dt%20%7B%0Afont%2Dweight%3A%20bold%3B%0A%7D%0Aul%20%7B%0Amargin%2Dtop%3A%200%3B%0A%7D%0Aul%20li%20%7B%0Alist%2Dstyle%3A%20circle%20outside%3B%0A%7D%0Aul%20ul%20%7B%0Amargin%2Dbottom%3A%200%3B%0A%7D%0Apre%2C%20code%20%7B%0Abackground%2Dcolor%3A%20%23f7f7f7%3B%0Aborder%2Dradius%3A%203px%3B%0Acolor%3A%20%23333%3B%0Awhite%2Dspace%3A%20pre%2Dwrap%3B%20%0A%7D%0Apre%20%7B%0Aborder%2Dradius%3A%203px%3B%0Amargin%3A%205px%200px%2010px%200px%3B%0Apadding%3A%2010px%3B%0A%7D%0Apre%3Anot%28%5Bclass%5D%29%20%7B%0Abackground%2Dcolor%3A%20%23f7f7f7%3B%0A%7D%0Acode%20%7B%0Afont%2Dfamily%3A%20Consolas%2C%20Monaco%2C%20%27Courier%20New%27%2C%20monospace%3B%0Afont%2Dsize%3A%2085%25%3B%0A%7D%0Ap%20%3E%20code%2C%20li%20%3E%20code%20%7B%0Apadding%3A%202px%200px%3B%0A%7D%0Adiv%2Efigure%20%7B%0Atext%2Dalign%3A%20center%3B%0A%7D%0Aimg%20%7B%0Abackground%2Dcolor%3A%20%23FFFFFF%3B%0Apadding%3A%202px%3B%0Aborder%3A%201px%20solid%20%23DDDDDD%3B%0Aborder%2Dradius%3A%203px%3B%0Aborder%3A%201px%20solid%20%23CCCCCC%3B%0Amargin%3A%200%205px%3B%0A%7D%0Ah1%20%7B%0Amargin%2Dtop%3A%200%3B%0Afont%2Dsize%3A%2035px%3B%0Aline%2Dheight%3A%2040px%3B%0A%7D%0Ah2%20%7B%0Aborder%2Dbottom%3A%204px%20solid%20%23f7f7f7%3B%0Apadding%2Dtop%3A%2010px%3B%0Apadding%2Dbottom%3A%202px%3B%0Afont%2Dsize%3A%20145%25%3B%0A%7D%0Ah3%20%7B%0Aborder%2Dbottom%3A%202px%20solid%20%23f7f7f7%3B%0Apadding%2Dtop%3A%2010px%3B%0Afont%2Dsize%3A%20120%25%3B%0A%7D%0Ah4%20%7B%0Aborder%2Dbottom%3A%201px%20solid%20%23f7f7f7%3B%0Amargin%2Dleft%3A%208px%3B%0Afont%2Dsize%3A%20105%25%3B%0A%7D%0Ah5%2C%20h6%20%7B%0Aborder%2Dbottom%3A%201px%20solid%20%23ccc%3B%0Afont%2Dsize%3A%20105%25%3B%0A%7D%0Aa%20%7B%0Acolor%3A%20%230033dd%3B%0Atext%2Ddecoration%3A%20none%3B%0A%7D%0Aa%3Ahover%20%7B%0Acolor%3A%20%236666ff%3B%20%7D%0Aa%3Avisited%20%7B%0Acolor%3A%20%23800080%3B%20%7D%0Aa%3Avisited%3Ahover%20%7B%0Acolor%3A%20%23BB00BB%3B%20%7D%0Aa%5Bhref%5E%3D%22http%3A%22%5D%20%7B%0Atext%2Ddecoration%3A%20underline%3B%20%7D%0Aa%5Bhref%5E%3D%22https%3A%22%5D%20%7B%0Atext%2Ddecoration%3A%20underline%3B%20%7D%0A%0Acode%20%3E%20span%2Ekw%20%7B%20color%3A%20%23555%3B%20font%2Dweight%3A%20bold%3B%20%7D%20%0Acode%20%3E%20span%2Edt%20%7B%20color%3A%20%23902000%3B%20%7D%20%0Acode%20%3E%20span%2Edv%20%7B%20color%3A%20%2340a070%3B%20%7D%20%0Acode%20%3E%20span%2Ebn%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Efl%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Ech%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Est%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Eco%20%7B%20color%3A%20%23888888%3B%20font%2Dstyle%3A%20italic%3B%20%7D%20%0Acode%20%3E%20span%2Eot%20%7B%20color%3A%20%23007020%3B%20%7D%20%0Acode%20%3E%20span%2Eal%20%7B%20color%3A%20%23ff0000%3B%20font%2Dweight%3A%20bold%3B%20%7D%20%0Acode%20%3E%20span%2Efu%20%7B%20color%3A%20%23900%3B%20font%2Dweight%3A%20bold%3B%20%7D%20%20code%20%3E%20span%2Eer%20%7B%20color%3A%20%23a61717%3B%20background%2Dcolor%3A%20%23e3d2d2%3B%20%7D%20%0A" rel="stylesheet" type="text/css" />

</head>

<body>




<h1 class="title toc-ignore">ProliferativeIndex Vignette</h1>
<h4 class="author"><em>Brittany N. Lasseigne, PhD and Ryne C. Ramaker</em></h4>
<h4 class="date"><em>2017-02-16</em></h4>



<p>The ProliferativeIndex R package<a href="#fn1" class="footnoteRef" id="fnref1"><sup>1</sup></a> provides users with R functions for calculating and analyzing the proliferative index (PI) from an RNA-seq dataset.</p>
<p>The PI was adapted from Venet, et al.<a href="#fn2" class="footnoteRef" id="fnref2"><sup>2</sup></a>:</p>
<blockquote>
<p>“The proliferating cell nuclear antigen, PCNA, is a ring-shaped protein that encircles DNA and regulates several processes leading to DNA replication. As suggested by its name, this is one of the most widely used antigen target for immunohistochemical measures of the fraction of proliferating cells in tissues. Ge et al. profiled with microarrays 36 tissues from normal, healthy individuals encompassing 27 organs. We call ‘meta-PCNA’ the signature composed of the 1% genes the most positively correlated with PCNA expression across these 36 tissues. In plain language, meta-PCNA genes are consistently expressed when PCNA is expressed in normal tissues and consistently repressed when PCNA is repressed. We define the meta-PCNA index as the median expressin of meta-PCNA genes.”</p>
</blockquote>
<p>IMPORTANT: Proliferative Indices are only interpretable relative to other PIs. For example, higher/lower PI in tumors compared to normal tissues or in post-mitotic tissues compared to in tissues with high rates of cell turnover. Additionally, PI is measuring proliferation associated with expression (as described above) and not necessarily proliferation itself.</p>
<p>ProliferativeIndex contains the following functions:</p>
<ul>
<li><code>readDataForPI</code> : Read in user data for use with package functions</li>
<li><code>calculatePI</code> : Calculate PI for user data</li>
<li><code>comparePI</code> : Compare PI across user data set</li>
<li><code>compareModeltoPI</code> : Compare PI to model PCs</li>
</ul>
<div id="example-data-set" class="section level2">
<h2>Example Data Set</h2>
<p>Included with ProliferativeIndex specifically for use with this vignette is data from the The Cancer Genome Atlas (TCGA) Adrenocortical Carcinoma (ACC) dataset.<a href="#fn3" class="footnoteRef" id="fnref3"><sup>3</sup></a></p>
<p>After first loading the ProliferativeIndex library:</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">library</span>(ProliferativeIndex)</code></pre></div>
<p>This dataset, <code>vstTCGA_ACCData_sub</code> can be accessed from the package:</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">data</span>(vstTCGA_ACCData_sub)

<span class="co">#Examine only the first few columns and rows because the dataset is large (20501 genes x 10 samples):</span>
<span class="kw">dim</span>(vstTCGA_ACCData_sub)</code></pre></div>
<pre><code>## [1] 20501    10</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="co">#Note that sample IDs are column names and HGNC gene IDs (http://www.genenames.org) are rownames and that vst data is numeric.</span>
<span class="kw">str</span>(vstTCGA_ACCData_sub)</code></pre></div>
<pre><code>## 'data.frame':    20501 obs. of  10 variables:
##  $ TCGA.OR.A5J1: num  5.87 4.19 5.92 8.43 6.99 ...
##  $ TCGA.OR.A5J2: num  5.49 4.19 5.2 8.74 4.19 ...
##  $ TCGA.OR.A5J3: num  6.04 4.52 5.44 8.04 4.76 ...
##  $ TCGA.OR.A5J5: num  11.4 4.71 5.22 7.08 6.8 ...
##  $ TCGA.OR.A5J6: num  10.07 4.19 5.11 8.8 4.66 ...
##  $ TCGA.OR.A5J7: num  5.57 4.19 4.96 7.52 4.91 ...
##  $ TCGA.OR.A5J8: num  6.86 4.19 4.19 6.91 5.1 ...
##  $ TCGA.OR.A5J9: num  5.4 4.19 6.46 8.94 6.34 ...
##  $ TCGA.OR.A5JA: num  6.8 4.19 5.25 8.77 6.36 ...
##  $ TCGA.OR.A5JB: num  8.53 4.19 4.19 6.84 4.19 ...</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">knitr::<span class="kw">kable</span>(vstTCGA_ACCData_sub[<span class="dv">1</span>:<span class="dv">5</span>,<span class="dv">1</span>:<span class="dv">5</span>])</code></pre></div>
<table>
<thead>
<tr class="header">
<th></th>
<th align="right">TCGA.OR.A5J1</th>
<th align="right">TCGA.OR.A5J2</th>
<th align="right">TCGA.OR.A5J3</th>
<th align="right">TCGA.OR.A5J5</th>
<th align="right">TCGA.OR.A5J6</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>A1BG</td>
<td align="right">5.871339</td>
<td align="right">5.490145</td>
<td align="right">6.036080</td>
<td align="right">11.397348</td>
<td align="right">10.065106</td>
</tr>
<tr class="even">
<td>A1CF</td>
<td align="right">4.190503</td>
<td align="right">4.190503</td>
<td align="right">4.523434</td>
<td align="right">4.713955</td>
<td align="right">4.190503</td>
</tr>
<tr class="odd">
<td>A2BP1</td>
<td align="right">5.915039</td>
<td align="right">5.196520</td>
<td align="right">5.443088</td>
<td align="right">5.221104</td>
<td align="right">5.112238</td>
</tr>
<tr class="even">
<td>A2LD1</td>
<td align="right">8.431843</td>
<td align="right">8.741279</td>
<td align="right">8.043286</td>
<td align="right">7.075708</td>
<td align="right">8.798831</td>
</tr>
<tr class="odd">
<td>A2ML1</td>
<td align="right">6.986670</td>
<td align="right">4.190503</td>
<td align="right">4.764641</td>
<td align="right">6.798125</td>
<td align="right">4.657211</td>
</tr>
</tbody>
</table>
</div>
<div id="readdataforpi-function" class="section level2">
<h2>readDataForPI function</h2>
<p>Functions in the ProliferativeIndex package come with help pages that can be accessed as usual (for example, <code>?readDataForPI</code>).</p>
<p>The function <code>readDataForPI</code> is used to read data in for use with the ProliferativeIndex package.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="co">#Inputs are the user's vst dataframe and a model of interest for examining PI:</span>
exampleTCGAData&lt;-<span class="kw">readDataForPI</span>(vstTCGA_ACCData_sub, <span class="kw">c</span>(<span class="st">&quot;AIFM3&quot;</span>, <span class="st">&quot;ATP9B&quot;</span>, <span class="st">&quot;CTRC&quot;</span>, <span class="st">&quot;MCL1&quot;</span>, <span class="st">&quot;MGAT4B&quot;</span>, <span class="st">&quot;ODF2L&quot;</span>, <span class="st">&quot;SNORA65&quot;</span>, <span class="st">&quot;TPPP2&quot;</span>))

<span class="co">#examine output which is a list of two objects:</span>
<span class="co"># exampleTCGAData$vstData is the user's vst dataframe and exampleTCGAData$modelIDs is a character string of the user's gene IDs for their model of interest</span>
<span class="kw">str</span>(exampleTCGAData)</code></pre></div>
<pre><code>## List of 2
##  $ vstData :'data.frame':    20501 obs. of  10 variables:
##   ..$ TCGA.OR.A5J1: num [1:20501] 5.87 4.19 5.92 8.43 6.99 ...
##   ..$ TCGA.OR.A5J2: num [1:20501] 5.49 4.19 5.2 8.74 4.19 ...
##   ..$ TCGA.OR.A5J3: num [1:20501] 6.04 4.52 5.44 8.04 4.76 ...
##   ..$ TCGA.OR.A5J5: num [1:20501] 11.4 4.71 5.22 7.08 6.8 ...
##   ..$ TCGA.OR.A5J6: num [1:20501] 10.07 4.19 5.11 8.8 4.66 ...
##   ..$ TCGA.OR.A5J7: num [1:20501] 5.57 4.19 4.96 7.52 4.91 ...
##   ..$ TCGA.OR.A5J8: num [1:20501] 6.86 4.19 4.19 6.91 5.1 ...
##   ..$ TCGA.OR.A5J9: num [1:20501] 5.4 4.19 6.46 8.94 6.34 ...
##   ..$ TCGA.OR.A5JA: num [1:20501] 6.8 4.19 5.25 8.77 6.36 ...
##   ..$ TCGA.OR.A5JB: num [1:20501] 8.53 4.19 4.19 6.84 4.19 ...
##  $ modelIDs: chr [1:8] &quot;AIFM3&quot; &quot;ATP9B&quot; &quot;CTRC&quot; &quot;MCL1&quot; ...</code></pre>
<p>*note, the R package includes a data object, ‘exReadDataObj’ that is the output from the readDataForPI function for comparison</p>
<div id="calculatepi-function" class="section level3">
<h3>calculatePI function</h3>
<p>The function <code>calculatePI</code> calculates PI for all sample’s in the users vst dataframe using a list of PCNA-associated genes collected from Venet et al. (including alternative gene names).</p>
<p>*note, the function will print to the screen how many genes used to calculate the PI were found in the vstData</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">proliferativeIndices&lt;-<span class="kw">calculatePI</span>(exampleTCGAData)</code></pre></div>
<pre><code>## [1] &quot;vstData contained 131/131 of the PI-associated genes&quot;</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">summary</span>(proliferativeIndices)</code></pre></div>
<pre><code>##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##   8.005   8.097   8.130   8.152   8.179   8.343</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="co">#Examine function output:</span>
knitr::<span class="kw">kable</span>(<span class="kw">head</span>(proliferativeIndices))</code></pre></div>
<table style="width:33%;">
<colgroup>
<col width="20%"></col>
<col width="12%"></col>
</colgroup>
<thead>
<tr class="header">
<th align="left">TCGA.OR.A5J1 TCGA.OR.A5J2 TCGA.OR.A5J3 TCGA.OR.A5J5 TCGA.OR.A5J6 TCGA.OR.A5J7</th>
<th align="right">8.005387 8.190936 8.131733 8.342858 8.084313 8.126670</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">*note, the R pa</td>
<td align="right">ckage includes a data object, ‘exVSTPI’ that is the output from the calculatePI function for comparison</td>
</tr>
<tr class="even">
<td align="left">## comparePI fu This function w</td>
<td align="right">nction ill summarize the PI values within the user’s dataset.</td>
</tr>
<tr class="odd">
<td align="left">Min. 1st Qu. 8.005 8.097 [](Proliferati *note, the R pa</td>
<td align="right">Median Mean 3rd Qu. Max. 8.130 8.152 8.179 8.343 veIndexVignette_files/figure-html/unnamed-chunk-5-1.png)<!-- --> ckage includes a data object, ‘exVSTPI’ that is the output from the calculatePI function for comparison</td>
</tr>
<tr class="even">
<td align="left">## compareModel</td>
<td align="right">toPI function</td>
</tr>
<tr class="odd">
<td align="left">The function `c</td>
<td align="right">ompareModeltoPI` will take, as input, the user’s data and model identifiers and compare to PI:</td>
</tr>
<tr class="even">
<td align="left"><code>r modelComparison</code></td>
<td align="right">&lt;-compareModeltoPI(exampleTCGAData, proliferativeIndices)</td>
</tr>
<tr class="odd">
<td align="left">[](Proliferati</td>
<td align="right">veIndexVignette_files/figure-html/unnamed-chunk-6-1.png)<!-- --></td>
</tr>
<tr class="even">
<td align="left"><code>r #the output is knitr::kable(mo</code></td>
<td align="right">a table, inspect: delComparison)</td>
</tr>
<tr class="odd">
<td align="left">Spearma</td>
<td align="right">nRho SpearmanPvalue PCAPropOfVariance</td>
</tr>
</tbody>
</table>
<p>PC1 0.3818182 0.2789652 0.51527 PC2 0.0424242 0.9186333 0.11587 PC3 0.2242424 0.5366881 0.07491 PC4 0.1272727 0.7328868 0.06558 PC5 -0.3575758 0.3128003 0.05897 PC6 -0.5515152 0.1042978 0.05068 PC7 0.3818182 0.2789652 0.05002 PC8 0.0060606 1.0000000 0.03992 PC9 0.1636364 0.6567214 0.02878 PC10 0.2727273 0.4482722 0.00000</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Ramaker and Lasseigne, et al. bioRxiv, 2016.<a href="#fnref1">↩</a></p></li>
<li id="fn2"><p>Venet, et al. PLoS Computational Biology, 2011 and Ge, et al. Genomics, 2005.<a href="#fnref2">↩</a></p></li>
<li id="fn3"><p>The TCGA ACC dataset was obtained from the TCGA data portal (tcga-data.nci.nih.gov) in June 2015. Level 3 RNASeqV2 raw count data was variance stabalized with the DESeq2 v1.8.2 ‘varianceStabilizingTransformation’.<a href="#fnref3">↩</a></p></li>
</ol>
</div>



<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
